Frameworks/CoreGraphics/CGBitmapImage.mm

```diff
@@ -452,7 +454,7 @@
         BYTE* imageData = static_cast<BYTE*>(LockImageData());
         ComPtr<IWICBitmap> wicBitmap = Make<CGIWICBitmap>(this, SurfaceFormat());
         ComPtr<ID2D1Factory> d2dFactory;
-        THROW_IF_FAILED(D2D1CreateFactory(D2D1_FACTORY_TYPE_SINGLE_THREADED, __uuidof(ID2D1Factory), &d2dFactory));
+        FAIL_FAST_IF_FAILED(_CGGetD2DFactory(&d2dFactory));
```

Comment:
Please no, THROW_IF_FAILED() is marginally better.  FAIL_FAST is essentially abort, and as framework we should seldom be using it.  And in other places too, per our offline discussion.